code-samples/eventing/bookstore-sample-app/db/README.md

```diff
@@ -1,26 +1,109 @@
-# Bookstore Database
+# Database Service for Bookstore
+To successfully launch the Bookstore sample application, it's essential to set up a dedicated database populated with specific sample data. This guide provides both the schema for the database and the initial data you'll need to get started.
 
-1. Database Schema
-2. Sample Data
+In this tutorial, we'll embark on creating a PostgreSQL database using Kubernetes (K8s) StatefulSets and populating it with the sample data provided.
 
-## 1. Database Schema
+You might wonder, "Why not leverage Knative Serving to dynamically scale the database service in response to traffic demands?" We'll delve into the optimal scenarios for employing Knative Serving and when it's advantageous for our database service.
 
-### BookReviews Table
+## What Knative features will we learn about?
+- Appropriate Use Cases for Knative Service

```

Comment:
I feel like the sentence and the point list is out of context and eventually contextualized below in `Why did we choose to deploy our PostgreSQL database using a StatefulSet instead of a Knative Service?`.

I think the section below is enough